flow-nodes/api-builder-plugin-logger/test/test.js

```diff
@@ -16,47 +22,55 @@ describe('flow-node logger', () => {
 		it('should define flow-nodes', () => {
 			expect(actions).to.be.an('object');
 			expect(actions.log).to.be.a('function');
-			expect(runtime).to.exist;
-			const flownode = runtime.getFlowNode('logger');
-			expect(flownode).to.be.a('object');
+			expect(plugin).to.be.a('object');
+			expect(plugin.getFlowNodeIds()).to.deep.equal([
+				'logger'
+			]);
+			expect(flowNode).to.be.a('object');
 
 			// Ensure the flow-node matches the spec
-			expect(flownode.name).to.equal('Logger');
-			expect(flownode.description).to.equal('Logger utility.');
-			expect(flownode.category).to.equal('general');
-			expect(flownode.icon).to.be.a('string');
-			expect(flownode.methods).to.deep.equal({
+			expect(flowNode.name).to.equal('Logger');
+			expect(flowNode.description).to.equal('Logger utility.');
+			expect(flowNode.category).to.equal('general');
+			expect(flowNode.icon).to.be.a('string');
+			expect(flowNode.methods).to.deep.equal({
 				log: {
+					name: 'Output Log',
 					description: 'Writes to the output log.',
 					parameters: {
 						level: {
 							description: 'The log level to write to.',
 							required: false,
 							initialType: 'string',
 							schema: {
+								title: 'Loglevel',
 								type: 'string',
 								default: 'trace',
-								enum: [ 'trace', 'debug', 'warn', 'error', 'time', 'timeEnd' ]
```

Comment:
I guess we removed 'time' and 'timeEnd' because they are not part of the logger and they never worked?  If they were used, I presume it would actually kill the flow?

Reply:
Yup that's right. They were there in my initial implementation since it was based on console, but after switching to the api builder logger they became invalid.

Reply:
@deepakvk, there is a unit test failing because of this.